src/phoenix/server/api/mutations/dataset_mutations.py

```diff
@@ -406,6 +517,33 @@ def _span_attribute(semconv: str) -> Any:
     return attribute_value.label(semconv.replace(".", "_"))
 
 
+def _to_orm_revision(
+    *,
+    previous_revision: models.DatasetExampleRevision,
```

Comment:
note: I think this confused me a bit as it's not previous yet - maybe existing_revision might be a better word here?

Reply:
Addressed in 86b575b5d2a4fc050452ab608c0fb0cea7a1fb99